test/specs/plugins.js

```diff
@@ -3,7 +3,7 @@ var test = require('tap').test
 var plugins = require('../../src/lib/plugins')
 
 test('export plugins', function (t) {
-  t.plan(5)
+  t.plan(6)
```

Comment:
Because this test block does not run multiple actions asynchronously, I would just remove the `t.plan` call, and call `t.end()` after line 15. Much less flaky (As you don't have to remember to increment the plan count each time you add a test.